### PR TITLE
Force single geom

### PR DESCRIPTION
**Pull request by popkinj**
Forcing only one geometry at a time.

### CODE REVIEW

File: app/src/components/map/MapContainer.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+#map {
+  height: 100%;
+  width: 100%;
+}
+
+.MuiContainer-root {
+  padding: 0 !important;
+}
+
```
Comment (NickPhura) [inline app/src/components/map/MapContainer.css]: Can you strip out this css? I tested it out, and It doesn't appear to be needed

All of the general container type css is being controlled by a parent component, and then passed down.  The goal is to slowly build up a single css theme object (got the bare bones going currently), that can then be referenced everywhere, so the app pages all have the same look but without having to maintain css in multiple places.

**Review by NickPhura — DISMISSED** [review]
Just had the one comment about the extra css.

Otherwise looks good!

File: app/src/components/map/MapContainer.css (hunk 1 above)
Comment (rstens) [reply app/src/components/map/MapContainer.css]: Failing build: 
```
react-scripts] H:/invasivesbc/app/src/App.tsx
[react-scripts] TypeScript error in H:/invasivesbc/app/src/App.tsx(5,48):
[react-scripts] Cannot find module 'contexts/DatabaseChangesContext'.  TS2307
[react-scripts]     3 | import { CircularProgress, makeStyles } from '@material-ui/core';
[react-scripts]     4 | import { KeycloakProvider } from '@react-keycloak/web';
[react-scripts]   > 5 | import { DatabaseChangesContextProvider } from 'contexts/DatabaseChangesContext';
[react-scripts]       |                                                ^
[react-scripts]     6 | import Keycloak, { KeycloakConfig, KeycloakInstance } from 'keycloak-js';
[react-scripts]     7 | import React from 'react';
[react-scripts]     8 | import AppRouter from './AppRouter';
```

Comment (popkinj) [reply app/src/components/map/MapContainer.css]: Right... I asked about this earlier in Teams. This css is needed for the map. And I'll need to add further in the future. Is there a better way to add css to the map?

Comment (popkinj) [reply app/src/components/map/MapContainer.css]: Thanks for tracking that down @rstens . But I don't think this PR touches that file.

**Review by popkinj — COMMENTED** [review]
Just removed the offending code that @rstens pointed out.

**Review by NickPhura — APPROVED** [review]

**Pull request merged**